### ISSUE TITLE
GameGuru MAX - Stand Alone Includes Tons of Files Not Needed!

### ISSUE THREAD

**Issue by ArgentArts** [TheGameCreators/GameGuruRepo#2301]
I exported a stand alone of a test file that contains a warehouse placed in an empty level (no "terrain" with no grass and no trees) and several my own PBR spheres for testing PBR materials. That was all that's in the level. But the folder is 1.34GB in size. Looking in the folder I see that there's a lot in there that is not a part of my test level. For example, every single particle that MAX has available to it is in the Files>particlebank folder. If there's not a single particle being used, why fill this folder with particles? My test level has no weapons, no characters, no zombies. Yet the audiobank folder is filled with audio files for bullet sounds, breath sounds, etc. All the footstep sounds are there for every terrain type even through there's no terrain in my level ... only the warehouse. Worse, is the terraintextures folder. What's up with that? Over 400MB of materials when I don't have any terrain in my level. I plan on creating an interior-only game. Why would I want to ship it with over 400MB of terrain materials? And what about the 137MB of tree billboards and textures in treebank (there's also stuff in vegbank)? Grassbank adds another 65MB. Best I can tell, my actual test level content (gamecore, entitybank, mapbank, etc.) clocks in at just under 600MB. So, over half of what's in this folder seems to be waste ... not needed to run the game. 

In fact, I just did a test. I deleted all those extra folders ... cutting my folder size down to just around 600MB. The test level ran perfectly without all the extra stuff. And you know this issue here:

https://github.com/TheGameCreators/GameGuruRepo/issues/2300

It didn't happen this time. If I run the test game with all the extra stuff in it, I get all the initial stuttering (as per the report linked above). Removed, it all goes away. The test level runs perfectly.

MAX needs to be smarter than this when it creates its stand alone and, to the best of its ability, only include what the game needs.

**Comment by ZakJudges** [comment]
Note: assimp.dll should not be needed for standalone

**Comment by LeeBamberTGC** [comment]
@ZakJudges Indeed, this DLL is not used, but it is hard-linked to the executable stub so removing it will instantly cause the executable to not find one of its dependencies and fail to launch.  Leaving this as a bug however as there are MANY other files that are copied by folder rather than as specifically detected as used, so there is some room for further improvement. If anyone else wants to contribute to this issue with other scenarios not mentioned in the original post that can be trimmed from the exported files, please do comment.

**Comment by MonkeyFrogStudio** [comment]
@LeeBamberTGC This is still an issue. Tons of unnecessary files are saved with the stand alone, bloating the size of the stand alone. 

Also, this is now an issue with stand alone games:

https://github.com/TheGameCreators/GameGuruRepo/issues/3515

Also the initial stutter (super-low frame rate) until you move the player is back:

https://github.com/TheGameCreators/GameGuruRepo/issues/2300

**Comment by LeeBamberTGC** [comment]
THIS IS THE NEW HOME FOR THE NEW PROJECT SYSTEM, a feature described in the attached PDF document.
[project_system.pdf](https://github.com/TheGameCreators/GameGuruRepo/files/15139848/project_system.pdf)
CLOSING ALL PARALLEL ISSUES AND REDIRECTING TO THIS CENTRAL ONE.
